rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -6,125 +6,146 @@ import com.cibo.evilplot.plot._
 import com.cibo.evilplot.numeric._
 import com.cibo.evilplot.colors._
 import com.cibo.evilplot.plot.renderers._
-import com.cibo.evilplot.plot.aesthetics.DefaultTheme._
+import com.cibo.evilplot.plot.aesthetics._
 
 object Jupyter {
-  implicit val extent = Extent(400, 400)
+  val font =
+    java.awt.GraphicsEnvironment.getLocalGraphicsEnvironment.getAvailableFontFamilyNames
+      .find(_.startsWith("Century Schoolbook"))
+      .getOrElse("Arial")
 
-  def traces(out: Seq[Map[String, Double]],
-             truth: Map[String, Double] = Map(),
-             lagMax: Int = 40,
-             numBars: Int = 50)(implicit outputHandler: OutputHandler): Unit = {
-    implicit val extent = Extent(1200, out.head.keys.size * 300.0)
-    show(EvilTracePlot.traces(out, truth, lagMax, numBars))
-  }
+  implicit val extent = Extent(400, 400)
+  implicit val theme = Theme(
+    fonts = DefaultTheme.DefaultFonts.copy(
+      tickLabelSize = 11,
+      labelSize = 12,
+      fontFace = font
+    ),
+    colors = DefaultTheme.DefaultColors.copy(
+      point = HSLA(211, 38, 48, 0.5),
+      fill = HSLA(210, 100, 0, 0.2)
+    ),
+    elements = DefaultTheme.DefaultElements.copy(
+      strokeWidth = 0.5,
+      pointSize = 3
+    )
+  )
 
-  def pairs(out: Seq[Map[String, Double]],
-            truth: Map[String, Double] = Map(),
-            numBars: Int = 30)(implicit outputHandler: OutputHandler): Unit = {
-    implicit val extent =
-      Extent(out.head.keys.size * 300.0, out.head.keys.size * 300.0)
-    show(EvilTracePlot.pairs(out, truth, numBars))
-  }
+  def density[N](seq: Seq[N], minX: Double, maxX: Double)(
+      implicit num: Numeric[N],
+      theme: Theme): Plot =
+    density(seq, "", Some(Bounds(minX, maxX)))
 
-  def histogram[N](seq: Seq[N], numBars: Int = 40)(implicit num: Numeric[N],
-                                                   oh: OutputHandler,
-                                                   extent: Extent): Unit = {
-    show(
-      Histogram(seq.map { n =>
-        num.toDouble(n)
-      }, numBars)
-        .xAxis()
-        .yAxis()
-        .frame()
-        .xLabel("x")
-        .yLabel("Frequency"))
-  }
+  def density[N](seq: Seq[N],
+                 xLabel: String = "",
+                 bounds: Option[Bounds] = None)(implicit num: Numeric[N],
+                                                theme: Theme): Plot =
+    Histogram(seq.map { n =>
+      num.toDouble(n)
+    }, binningFunction = Histogram.density, xbounds = bounds)
+      .xLabel(xLabel)
```

Comment:
This shouldn't set xlabel/yabel/xaxis/yaxis/frame anymore right?

Reply:
Yeah, fixed (and added a `show` variant that only does an x-axis to get around the need to write "Density" in every time)